src/components/Modal/ModalButtons/ModalButtons.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { FC } from "react";
+import VibeComponentProps from "../../../types/VibeComponentProps";
+import Button from "../../Button/Button";
+import Flex from "../../Flex/Flex";
+import { ComponentDefaultTestId } from "../../../tests/constants";
+import { getTestId } from "../../../tests/test-ids-utils";
+
+interface ModalButtonsProps extends VibeComponentProps {
+  className: string;
```

Comment:
It's used to be internal component only for docs, But I guess we want alignment between all instance appearance of the modal footer buttons

Reply:
this should not be here, should be in Atmosphere please remove